README.md

```diff
@@ -1,5 +1,7 @@
 # TrixiParticles.jl
 
+[![Docs-stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://trixi-framework.github.io/TrixiParticles.jl/stable)
+[![Docs-dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://trixi-framework.github.io/TrixiParticles.jl/dev)
```

Comment:
I also added the docs badges. Note that the first one doesn't work yet, but will (hopefully) work in 2 days when the package is registered.